p2p/peer.py

```diff
@@ -9,6 +9,7 @@
     ABC,
     abstractmethod
 )
+from datetime import datetime
```

Comment:
Can you adjust this to just be `import datetime`

see https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#dont-use-from-datetime-import-datetime for the why.